src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -309,19 +324,48 @@ const InvoiceForm = ({
                           />
                         </KeyValue>
                       </Col>
+                    </Row>
+
+                    <Row>
                       <Col data-test-col-lock-total xs={3}>
-                        <Field
-                          component={Checkbox}
+                        <Checkbox
+                          checked={isManualAmountEnabled}
+                          data-testid="lock-total"
                           disabled={isEditPostApproval}
+                          id="lock-total"
                           label={<FormattedMessage id="ui-invoice.invoice.lockTotal" />}
-                          name="lockTotal"
-                          type="checkbox"
+                          onChange={enableManualAmount}
                           vertical
                         />
                       </Col>
-                    </Row>
-
-                    <Row>
+                      <Col data-test-col-manual-amount xs={3}>
+                        {isEditPostApproval
```

Comment:
Isn't it handled inside of `TextField` with `isNonInteractive` prop?

Reply:
Removed `isNonInteractive ` prop since we need to display amounts with currency sign